integration-tests/test/ovm-self-upgrades.spec.ts

```diff
@@ -0,0 +1,202 @@
+import { expect } from 'chai'
+import { Wallet, utils, BigNumber, Contract, ContractFactory } from 'ethers'
+
+import { OptimismEnv } from './shared/env'
+
+import { getContractInterface } from '@eth-optimism/contracts'
+import { l2Provider, OVM_ETH_ADDRESS } from './shared/utils'
+import { ethers } from 'hardhat'
+
+// TODO: use actual imported Chugsplash type
```

Comment:
You should be able to get these types from `@eth-optimism/conracts` now (`ChugSplashAction`).